plugins/processors/starlark/starlark_test.go

```diff
@@ -113,7 +113,7 @@ def apply(metric):
 					time.Unix(0, 0),
 				),
 			},
-			expected: []telegraf.Metric{},
+			expected: make([]telegraf.Metric, 0),
```

Comment:
Remove, same for the other instances in here.

Reply:
Removed.